### PR TITLE
feat: Add flick_uia2

### PR DESCRIPTION
**Pull request by ki4070ma**
### Background
For https://github.com/appium/python-client/issues/512

### TODO
- [x] unit test
- [x] func test

### CODE REVIEW

File: appium/webdriver/webdriver.py [hunk 1]
```diff
@@ -345,6 +345,50 @@ def set_value(self, element: MobileWebElement, value: str) -> T:
...
         return self
 
+    def flick_element(self, element: Optional[MobileWebElement] = None,
+                      xspeed: int = 0, yspeed: int = 0,
+                      xoffset: int = 0, yoffset: int = 0,
+                      speed: int = 0) -> T:
+        """Flick the element
+
+        Args:
+            element: the element to be flicked
```
Comment (ki4070ma) [inline appium/webdriver/webdriver.py:355]: [Notes] https://github.com/appium/appium-uiautomator2-server/blob/c0d2d45740a2f9d05cf0e6c3b7b1a19e2f85d1b1/app/src/main/java/io/appium/uiautomator2/handler/Flick.java#L52

Comment (mykola-mokhnach) [reply appium/webdriver/webdriver.py:355]: these shortcuts were deliberately removed from java client as they are not cross-platform. 
I would rather prefer splitting platform-specific actions and assigning them to their appropriate driver classes

Comment (ki4070ma) [reply appium/webdriver/webdriver.py:355]: thanks, I haven't taken care of `not cross-platform` (uia2).

**Comment by KazuCocoa** [comment]
Selenium Python binding has the command, btw.
https://github.com/SeleniumHQ/selenium/blob/941dc9c6b2e2aa4f701c1b72be8de03d4b7e996a/py/selenium/webdriver/common/touch_actions.py#L168
You can simply add the same method in this touch action module

**Pull request closed without merging**